docs/features/wait/introduction.md

```diff
@@ -15,6 +15,7 @@ Below you can find a list of the available wait strategies that you can use:
 - [Log](./log.md)
 - [Multi](./multi.md)
 - [SQL](./sql.md)
+- [TLS](./tls.md)
```

Comment:
Oh sorry, I was not clear, we also need to add it here: https://github.com/testcontainers/testcontainers-go/blob/main/mkdocs.yml#L56